docs/cow-protocol/concepts/introduction/batch-auctions.md

```diff
@@ -3,9 +3,10 @@ sidebar_position: 3
 ---
 
 # Batch auctions
+Continuous limit order book designs leave room for arbitrage, incentivizing high-frequency trading and resulting in sub-optimal pricing for traders. Batch auctions are a new type of trading mechanism which addresses these flaws by grouping orders into discrete time intervals. This approach eliminates the arbitrage opportunities inherent in continuous limit order book designs and delivers more beneficial outcomes for traders. 
```

Comment:
Not sure if this is quite true. Perhaps @acanidio-econ / @harisang could weigh in if this is structurally correct?

Reply:
Took 3 months.... Makes sense to me, but let's wait for @acanidio-econ to confirm.